subsys/bluetooth/host/conn.c

```diff
@@ -2884,6 +2884,66 @@ int bt_conn_le_get_tx_power_level(struct bt_conn *conn,
 	return err;
 }
 
+#if defined(CONFIG_BT_PATH_LOSS_MONITORING)
+void notify_path_loss_threshold_report(struct bt_conn *conn,
+				       struct bt_conn_le_path_loss_threshold_report report)
+{
+	for (struct bt_conn_cb *cb = callback_list; cb; cb = cb->_next) {
+		if (cb->path_loss_threshold_report) {
+			cb->path_loss_threshold_report(conn, &report);
+		}
+	}
+
+	STRUCT_SECTION_FOREACH(bt_conn_cb, cb)
+	{
+		if (cb->path_loss_threshold_report) {
+			cb->path_loss_threshold_report(conn, &report);
+		}
+	}
+}
+
+int bt_conn_le_set_path_loss_monitoring_parameters(struct bt_conn *conn,
+				const struct bt_conn_le_path_loss_reporting_parameters *params)
+{
+	struct bt_hci_cp_le_set_path_loss_reporting_parameters *cp;
+	struct net_buf *buf;
+
+	buf = bt_hci_cmd_create(BT_HCI_OP_LE_SET_PATH_LOSS_REPORTING_PARAMETERS, sizeof(*cp));
+	if (!buf) {
+		return -ENOBUFS;
+	}
+
+	cp = net_buf_add(buf, sizeof(*cp));
+	cp->handle = sys_cpu_to_le16(conn->handle);
+	cp->high_threshold = params->high_threshold;
+	cp->high_hysteresis = params->high_hysteresis;
+	cp->low_threshold = params->low_threshold;
+	cp->low_hysteresis = params->low_hysteresis;
+	cp->min_time_spent = params->min_time_spent;
```

Comment:
This is 16 bits so needs endianness conversion as well.
```suggestion
	cp->min_time_spent = sys_cpu_to_le16(params->min_time_spent);
```